### PR TITLE
Adds new documentation for IsInt validator

### PR DESCRIPTION
**Pull request by froschdesign**
|    Q          |   A
|-------------- | ------
| Documentation | yes

### CODE REVIEW

File: docs/book/validators/is-int.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+Integer values are often written differently based on country or region. For
+example, using English, you may write `1234` or `1,234`; both are integer
+values, but the grouping is optional. In German, you'd write `1.234`, and in
+French, `1 234`.
+
+`Laminas\I18n\Validator\IsInt` will use a provided locale when evaluating the
+validity of an integer value. In such cases, it doesn't simply strip the
+separator, but instead validates that the correct separator as defined by the
+locale is used.
```
Comment (michalbundyra) [inline docs/book/validators/is-int.md]: I might be wrong, but something sounds to be wrong for me: "as defined by used locale"?

File: docs/book/validators/is-int.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+```php
+$validator = new Laminas\I18n\Validator\IsInt();
+
+$validator->isValid(1234);    // returns true
+$validator->isValid(1234.5);  // returns false
+$validator->isValid('1,234'); // returns true
+```
+
+By default, if no locale is provided, `IsInt` will use the system locale
+provide by PHP's `Locale` class and the `getDefault()` method.
```
Comment (michalbundyra) [inline docs/book/validators/is-int.md]: "provided by PHP's" - missing "d" in "provided"

**Comment by michalbundyra** [comment]
Thanks, @froschdesign!

**Comment by froschdesign** [comment]
@michalbundyra 
Thanks for your quick help and review on Sunday! 👍

**Pull request closed without merging**